packages/cli/src/generators/pull.js

```diff
@@ -0,0 +1,54 @@
+const colors = require('colors/safe');
+const debug = require('debug')('zapier:pull');
+const inquirer = require('inquirer');
+const path = require('path');
+const Generator = require('yeoman-generator');
+
+const maybeOverwriteFiles = async (gen) => {
+  const dstDir = gen.options.dstDir;
+  const srcDir = gen.options.srcDir;
+  for (const file of gen.options.sourceFiles) {
+    gen.fs.copyTpl(
+      gen.templatePath(path.join(srcDir, file)),
+      gen.destinationPath(path.join(dstDir, file)),
+      gen.options
+    );
+  }
+};
+
+module.exports = class PullGenerator extends Generator {
+  initializing() {
+    debug('SRC', this.options.sourceFiles);
+    debug('DEL', this.options.deletableFiles);
+  }
+
+  prompting() {
+    const prompts = [
+      {
+        type: 'confirm',
+        name: 'confirm',
+        message: `Warning: You are about to overwrite existing files.
+
+Before proceeding, please make sure you have saved your work. Consider creating a backup or saving your current state in a git branch. During the process, you may abort anytime by pressing 'x'. 
```

Comment:
My editor found a trailing space:

```suggestion
Before proceeding, please make sure you have saved your work. Consider creating a backup or saving your current state in a git branch. During the process, you may abort anytime by pressing 'x'.
```